src/__tests__/Collection-test.js

```diff
@@ -10,19 +10,25 @@
 
 'use strict';
 
-jest.autoMockOff();
-
-var recast = require('recast');
-var Collection = require('../Collection');
-
-var NodePath = recast.types.NodePath;
-var types = recast.types.namedTypes;
-var b = recast.types.builders;
 
 describe('Collection API', function() {
   var nodes;
+  var Collection;
+  var recast;
+  var NodePath;
+  var types;
+  var b;
 
   beforeEach(function() {
+    jest.resetModuleRegistry();
```

Comment:
jest 15 no longer calls this by default. This is the only file where it is actually needed but I figured I would add it on the other files as well since this pattern is used so that modules are expected to be isolated.